src/mo_types/type.ml

```diff
@@ -851,18 +858,20 @@ let serializable allow_mut t =
       | Mut t -> allow_mut && go t
       | Con (c, ts) ->
         (match Cons.kind c with
-        | Abs _ -> false
+        | Abs (bind_list, _, _) ->
+          allow_stable_functions
```

Comment:
This means we consider all type parameters stable. Is that even sound? I wouldn't expect it to be.

Reply:
Yes, for stable functions, type parameters are implicitly stable.
For flexible functions not.
The type arguments are also checked to be stable for stable functions only.

I initially thought this would be visible restriction but didn't notice any in the test code so far. My reasoning is that lambdas for type arguments are relatively rare and if needed, a flexible function can do.